src/elements/fields/PaymentMethodField.jsx

```diff
@@ -0,0 +1,193 @@
+import React, { memo, useEffect } from 'react';
+import {
+  CardElement,
+  Elements,
+  useElements,
+  useStripe
+} from '@stripe/react-stripe-js';
+
+import { getStripePromise } from '../../integrations/stripe';
+import { filterStyleValues } from '../../utils/styles';
+import { filterKeys } from '../../utils/utils';
+
+import InlineTooltip from '../components/Tooltip';
+
+// In order for the stripe card element to operate and show the hybrid fields, stripe must be loaded and 
+// initialized with a key.  In a runtime form, it is crucial to use the real key.  However. in 
+// the dashboard any key will do since the field is not operational nor would we want it to be.
+const stripeKey = 'any-key-does-not-matter-what';
+const stripePromise = getStripePromise();
+// Limit stripe css props to only those that are supported in order to avoid easy to miss warnings
+const supportedStripeCssProps = [
+  'backgroundColor',
+  'color',
+  'fontFamily',
+  'fontSize',
+  'fontSmoothing',
+  'fontStyle',
+  'fontVariant',
+  'fontWeight',
+  'iconColor',
+  'lineHeight',
+  'letterSpacing',
+  'textAlign',
+  'padding',
+  'textDecoration',
+  'textShadow',
+  'textTransform'
+];
+
+const CardField = ({
+  element,
+  runtime,
+  applyStyles,
+  fieldLabel,
+  elementProps = {},
+  setCardElement = () => {},
+  setFieldError = () => {},
+  onChange = () => {},
+  editable = false,
+  inlineError,
+  errorDisplayMode
+}) => {
+  const stripe = useStripe();
+  const elements = useElements();
+
+  useEffect(() => {
+    if (!stripe || !elements) {
+      // Stripe.js has not yet loaded.
+      return;
+    }
+    setCardElement(element.servar.key, elements.getElement(CardElement));
+  }, [stripe, element.servar.key, elements]);
+
+  const cardElementOptions = {
+    style: {
+      base: {
+        textDecoration: 'none', // Bug in card element - force the reset to none
+        ...filterKeys(applyStyles.getTarget('field'), supportedStripeCssProps),
+        '::placeholder': {
+          textDecoration: 'none', // Bug in card element - force the reset to none
+          ...filterKeys(
+            applyStyles.getTarget('placeholder'),
+            supportedStripeCssProps
+          )
+        },
+        ':hover': filterStyleValues(
```

Comment:
instead of retroactively filtering out teh !important suffix, it might be cleaner to pass in a flag to omit the !important flag when generating the hover/selected styles for applyStyles

Reply:
Changed ApplyStyles.applySelectorStyles just a bit (with a unit test as well) so that the !important is parameterized.  Removed filterStyleValues() completely.